README.md

```diff
@@ -277,3 +277,10 @@ Getting cluster info and generating kubekey's configuration file (optional).
 * [Storage clients](docs/storage-client.md)
 * [Kubectl autocompletion](docs/kubectl-autocompletion.md)
 * [Roadmap](docs/roadmap.md)
+
+## Contribute
```

Comment:
It's good to have a noun instead of a verb here. See also https://github.com/kubernetes/ingress-nginx#contributing.

Reply:
By the way, you just forgot to sign off your commits.